ansible/roles/xroad-is/tasks/setup-xrd4j.yml

```diff
@@ -48,3 +48,21 @@
     state: started
     enabled: yes
     daemon_reload: yes
+
+- name: Download test-service
+  get_url:
+    url: "https://artifactory.niis.org/xroad-maven-snapshots/org/niis/test-service/0.0.4-SNAPSHOT/test-service-0.0.4-20250326.073629-1.war"
```

Comment:
Maybe take the latest version, like example adapter? I guess it is changing very rarely, but it changes someone should not forget to change it here.